AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -997,6 +1028,7 @@ - (void)rangeController:(ASRangeController *)rangeController didDeleteNodes:(NSA
     [_layoutFacilitator collectionViewWillEditCellsAtIndexPaths:indexPaths batched:NO];
     [UIView performWithoutAnimation:^{
       [super deleteItemsAtIndexPaths:indexPaths];
```

Comment:
Please check out my feedback on the last change, which described how move and reload operations can also trigger the need for a Batch request.  Is this maybe an incorrect assumption?

We should probably have a mechanism to defer checking for batching if currently handling a batch EDIT operations, until the commit / endEditing occurs.  It seems like having these checks in every added method may add up with some overhead unless we have a simple strategy for coalescing them. This is probably not essential at this stage but is worth thinking about.

The feedback on the earlier change also asked that we investigate a simpler way to include all of these calls, because I believe there will be at least eight, And described one approach we could use for this. This is not a blocker for the change landing, but it would be great to hear your thoughts on each area feedback written - so I know if perhaps I missed something and did not realize that the suggestion is not applicable at all, or have a sense if you would suggest that we wait until a future improvement of the code to adopted that change.